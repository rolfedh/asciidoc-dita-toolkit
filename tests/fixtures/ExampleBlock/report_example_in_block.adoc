// An unsupported example in a sidebar block:
****
A paragraph.

====
An unsupported example.
====
****

// An unsupported example in a delimited block:
--
A paragraph.

====
An unsupported example.
====
--
