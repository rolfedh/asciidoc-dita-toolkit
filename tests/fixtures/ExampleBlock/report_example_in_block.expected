// An unsupported example in a sidebar block:

====
An unsupported example.
====

****
A paragraph.
****

// An unsupported example in a delimited block:

====
An unsupported example.
====

--
A paragraph.
--
